core/templates/services/rte-helper-modal.controller.spec.ts

```diff
@@ -356,21 +356,22 @@ describe('RteHelperModalComponent', () => {
       );
       expect(component.isErrorMessageNonempty()).toBe(true);
       expect(component.errorMessage).toBe(
-        'Please ensure that each title and content are filled'
+        'Please ensure that tab 1 title is filled.'
```

Comment:
Please ensure that the title of tab 1 is filled

Reply:
Done!